website/docs/prysm-usage/fee-recipient.md

```diff
@@ -0,0 +1,76 @@
+---
+id: fee-recipient
+title: Fee Recipient
+sidebar_label: fee recipient
+---
+
+# Fee Recipient
+
+Fee Recipient is the feature that allows users receive gas fees when proposing blocks. These gas rewards are some times know as the inclusion rewards.
+
+:::tip Post Bellatrix Fork ( The Merge )
+This feature will only activate Post Bellatrix Fork. Currently stakers do not receice gas fees from proposing blocks.
+:::
+
+:::warning Known Release Bug
+Setting a default fee recipient through the validator client is currently not working in the upcoming branch a fix was created and merged into develop for the following release. please refer to [#10555](https://github.com/prysmaticlabs/prysm/pull/10555)
+:::
+
+:::warning Be aware of your Fee Recipient Settings!!
```

Comment:
```suggestion
:::warning Incorrectly configured Fee Recipient settings could lead to a loss of earnings.
```
If possible, detail exactly what the worst-case is - configuration issues here won't impact anything other than these new gas fee rewards, right? Even in the worst case?